Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Estimate.hs

```diff
@@ -117,10 +119,5 @@ mkAdditionalBreakups mkPrice mkBreakupItem distance farePolicy = do
       DFP.ProgressiveDetails det -> mkAdditionalProgressiveBreakups det
       DFP.SlabsDetails det -> mkAdditionalSlabBreakups $ DFP.findFPSlabsDetailsSlabByDistance distance det.slabs
 
-    mkAdditionalProgressiveBreakups det = do
```

Comment:
why did you remove this?

Reply:
> why did you remove this?

I can't put anything in EXTRA_PER_KM_FARE, it's not constant now, right?

Reply:
Yup. For now, just put the startDistance `0` value in this. This param is needed for backward compatibility. 
Later, we can think of creating new param for whole list of per km fares.